cloudformation.yaml

```diff
@@ -34,14 +36,9 @@ Resources:
     Type: 'AWS::ApiGateway::RestApi'
     Properties:
       BinaryMediaTypes:
+      # TODO: Remove this and use modeled JSON structures?
```

Comment:
We should cut issues for these so that we have a more visible reminder to go work on them.